docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -5290,6 +5290,64 @@ The gcs_backend block configures the connection to Google Cloud Storage object s
 # 3. On Google Compute Engine it fetches credentials from the metadata server.
 # CLI flag: -<prefix>.gcs.service-account
 [service_account: <string> | default = ""]
+
+http:
+  # (advanced) The time an idle connection will remain idle before closing.
+  # CLI flag: -<prefix>.gcs.http.idle-conn-timeout
+  [idle_conn_timeout: <duration> | default = 1m30s]
+
+  # (advanced) The amount of time the client will wait for a servers response
+  # headers.
+  # CLI flag: -<prefix>.gcs.http.response-header-timeout
+  [response_header_timeout: <duration> | default = 2m]
+
+  # (advanced) If the client connects to object storage via HTTPS and this
+  # option is enabled, the client will accept any certificate and hostname.
+  # CLI flag: -<prefix>.gcs.http.insecure-skip-verify
+  [insecure_skip_verify: <boolean> | default = false]
+
+  # (advanced) Maximum time to wait for a TLS handshake. 0 means no limit.
+  # CLI flag: -<prefix>.gcs.tls-handshake-timeout
+  [tls_handshake_timeout: <duration> | default = 10s]
+
+  # (advanced) The time to wait for a server's first response headers after
+  # fully writing the request headers if the request has an Expect header. 0 to
+  # send the request body immediately.
+  # CLI flag: -<prefix>.gcs.expect-continue-timeout
+  [expect_continue_timeout: <duration> | default = 1s]
+
+  # (advanced) Maximum number of idle (keep-alive) connections across all hosts.
+  # 0 means no limit.
+  # CLI flag: -<prefix>.gcs.max-idle-connections
+  [max_idle_connections: <int> | default = 100]
+
+  # (advanced) Maximum number of idle (keep-alive) connections to keep per-host.
+  # If 0, a built-in default value is used.
```

Comment:
It turns out the default comes from the Go stdlib and it's "2" at time of writing. 

I guess our options are:
* Keep the docs as-is, vague because this is something we don't control
* Mention that the default is 2. It's a public constant at least in the Go stdlib so it's knowable.

Reply:
Pushed changes the mention the default of `2`